.travis.yml

```
@@ -82,7 +83,8 @@ jobs:
 
       before_script:
         - npm run doc-prepare
-        - npm run --prefix doc/framework clone-repos
+        - if [[ $TRAVIS_BRANCH =~ "^master|[0-9]+-stable$" ]]; then export BRANCH=stable; else export BRANCH=dev; fi
```

NP: readability improvement:
```suggestion
        - |
        if [[ $TRAVIS_BRANCH =~ "^master|[0-9]+-stable$" ]] then 
          export BRANCH=stable
        else 
          export BRANCH=dev
        fi
```